source/_localCaptioner/panel.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: UTF-8 -*-
+# NVDA add-on: Caption Local
+# Copyright (C)
+# This file is covered by the GNU General Public License.
```

Comment:
please update to use our [copyright headers](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/copyrightHeaders.md)